app/views/hyrax/base/_form_warning.html.erb

```diff
@@ -14,14 +14,15 @@
 <ul>
   <li class="radio">
     <label>
-      <%= f.radio_button :content_alert, t('hyrax.content_warning.content_alert') %>
+      <% @checked = (f.object.content_alert.blank? || (f.object.content_alert == t('hyrax.content_warning.content_alert'))) %>
```

Comment:
Can you move the `@checked` definition out of the `<ul>` so it's clearer where it's going to be used?